README.md

```diff
@@ -45,15 +45,9 @@ Pkg.add(url="https://github.com/SnowflurrySDK/SnowflurryPlots.jl", rev="main")
 
 # Getting Started
 
-A typical workflow to execute a quantum circuit on a quantum service consists of these three steps.
+The best way to learn Snowflurry is to use it! Let's try to make a two-qubit circuit which implements a [Bell/EPR state](https://en.wikipedia.org/wiki/Bell_state). We'll use Snowflurry to construct and simulate the circuit then verify our construction.
```

Comment:
change "then verify our construction" to for instance "then verify the produced Ket". Or else it seems we are verify the construction of the `circuit` object, not the resulting `Ket`.